ambry-clustermap/src/test/java/com/github/ambry/clustermap/HelixParticipantTest.java

```diff
@@ -77,6 +79,7 @@ public HelixParticipantTest(String stateModelDef) {
     props.setProperty("clustermap.state.model.definition", stateModelDef);
     this.stateModelDef = stateModelDef;
     helixManagerFactory = new MockHelixManagerFactory();
+    accountService = Mockito.mock(AccountService.class);
```

Comment:
Seems never used, can be removed. (Also format this file)

Reply:
Removed.